index.html

```diff
@@ -62,7 +63,7 @@ <h1>
           <button name="next section"><i class="fa fa-angle-down" id="next-section"></i></button>
         </div>
       </section>
-
+      <div id="popUp-modal"></div>
       <section class="work-section" id="portfolio">
```

Comment:
- [ ] Good job working on the modal, but please kindly ensure the projects/works section is generated dynamically from the projects array used in generating the modal. So kindly remove the projects/works section from the html file and generate it using Javascript